service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
-[id="chap-using-servicediscovery"]
-=  Connecting {product-long} Service to your OpenShift cluster
-ifdef::context[:parent-context: {context}]
-:context: using-servicediscovery
+[id="chap-connecting-openshift-to-streams-for-apache-kafka_{context}"]
+= Connecting OpenShift applications to {product-long}
+:context: connecting-streams-openshift
 
 [IMPORTANT]
 ====
 {product-long} is currently available for Development Preview. Development Preview releases provide early access to a limited set of features that might not be fully tested and that might change in the final GA version. Users should not use Development Preview software in production or for business-critical workloads. Limited documentation is available for Development Preview releases and is typically focused on fundamental user goals.
 ====
 
-// Purpose statement for the assembly
 [role="_abstract"]
-As a developer of applications and services, you can connect Quarkus applications to Kafka instances in {product-long}. https://quarkus.io/[Quarkus^] is a Kubernetes-native Java framework made for Java virtual machines (JVMs) and native compilation, and optimized for serverless, cloud, and Kubernetes environments. Quarkus is designed to work with popular Java standards, frameworks, and libraries like Eclipse MicroProfile and Spring, as well as Apache Kafka, RESTEasy (JAX-RS), Hibernate ORM (JPA), Infinispan, Camel, and many more.
+As a developer of applications and services, there might be cases where you need to connect applications deployed on a Kubernetes platform such as Red Hat OpenShift to Kafka instances created in {product-long}.
 
-OpenShift is an Enteprise Kuberentes Platform. When deploying Quarkus and any other type of application, developers can use
-the https://developers.redhat.com/blog/2019/12/19/introducing-the-service-binding-operator/[Service Binding Operator] to
-enable your applications to discover service configuration automatically in your OpenShift or Kubernetes instance.
+For example, suppose that you have two applications deployed to an OpenShift cluster; one that publishes price updates for a variety of stocks, and another that consumes the price updates for publication on a web page. You also have a Kafka instance in {product-long}. Each time there is a price update, you want to use the Kafka instance to forward the update as an event to the consuming application. To achieve this, you need a way to connect the applications in your OpenShift cluster to your Kafka instance in {product-long}. Connecting a Kubernetes application to a backing service such as {product-long} is called __service binding__.
 
-You can review all possible commands in https://github.com/redhat-developer/app-services-cli/blob/main/docs/commands/rhoas_cluster.adoc[cli documentation].
+This guide describes how to use the OpenShift Application Services (RHOAS) Operator to perform service binding for {product-long}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
```

Comment:
I think that we introduce full name on the top and then we can use Application services or RHOAS.
It will be actually easier for us to do it once on top and then use shortcut - RHOAS. 
WDYT?

```suggestion
This guide describes how to use the Application Services (RHOAS) Operator to perform service binding for {product-long}. The Kubernetes platform used throughout this guide is Red Hat OpenShift.
```

Reply:
@wtrocki For modular documentation, we typically spell out an abbreviation in full when it it first used in an assembly or module. Therefore, the RHOAS abbreviation (including `OpenShift`) is spelled out a few times  - at the beginning of the guide, in the conceptual section about the Operator, and at the start of the tutorial. But we then use `RHOAS` in the remainder of those modules.